docker/docker/src/main/java/org/arquillian/cube/docker/impl/util/AutoStartOrderUtil.java

```diff
@@ -70,6 +86,14 @@
         return new HashSet<>(nodes.values());
     }
 
+    private static boolean isRegularExpressionBased(String[] autoStartContainers) {
+        return autoStartContainers.length == 1;
```

Comment:
Yes I thought this but I am not sure how to resolve this in a better way. Maybe adding that if starts with keyword `regexp:` then it is a regular expression. But I tried to be as much clear as possible. But maybe I need to add this approach.